### PR TITLE
Fetaure/footer reskin

### PR DESCRIPTION
**Pull request by dutraneto**

### CODE REVIEW

File: components/Footer/Footer.module.scss [hunk 1]
```diff
@@ -4,19 +4,17 @@
   }
 }
 .footerNav {
-  width: calc(100% - 30px);
```
Comment (marceloglacial) [inline components/Footer/Footer.module.scss:7]: Remove the scroll

**Review by marceloglacial — CHANGES_REQUESTED** [review]
Remove the scroll and the extra spacing on the end. 

Screen Shot 2022-06-20 at 5 30 20 PM

File: components/Footer/Footer.module.scss (hunk 1 above)
Comment (dutraneto) [reply components/Footer/Footer.module.scss:7]: Removed

Comment (marceloglacial) [reply components/Footer/Footer.module.scss:7]: Add spacing to the bottom to be equal to the horizontal spacing

**Review by marceloglacial — CHANGES_REQUESTED** [review]

**Comment by marceloglacial** [comment]
The build is failing.

File: components/Footer/Footer.module.scss (hunk 1 above)
Comment (dutraneto) [reply components/Footer/Footer.module.scss:7]: Done

File: components/Padrinho/Padrinho.module.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.ctaWrapper {
```
Comment (marceloglacial) [inline components/Padrinho/Padrinho.module.scss]: You don't need a new CSS wrapper for it. Use the same pattern of [slug] page

File: components/Footer/Footer.module.scss (hunk 1 above)
Comment (marceloglacial) [reply components/Footer/Footer.module.scss:7]: Good job

File: components/Footer/Footer.module.scss [hunk 3]
```diff
@@ -4,19 +4,17 @@
...
+  width: 100%;
   background: var(--green);
   text-align: center;
-  position: fixed;
-  bottom: 1rem;
-  left: 15px;
   border-radius: var(--border-radius);
-  z-index: 9999;
+  padding: 2rem;
```
Comment (marceloglacial) [inline components/Footer/Footer.module.scss]: Too many padding. Follow the copyright pattern. It's supposed to be 1rem.

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/Padrinho/Padrinho.module.scss (hunk 2 above)
Comment (dutraneto) [reply components/Padrinho/Padrinho.module.scss]: I'm afraid I'm not following you. Wasn't I supposed to create the Padrinho component, though?

Comment (marceloglacial) [reply components/Padrinho/Padrinho.module.scss]: The component, yes. But you don't need an extra css file/style for it.

File: components/Footer/Footer.module.scss (hunk 3 above)
Comment (dutraneto) [reply components/Footer/Footer.module.scss]: Done

**Review by marceloglacial — APPROVED** [review]
Good job

**Comment by dutraneto** [comment]
Morning! Could you please, release the merging? Thank you!

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**